index.js

```diff
@@ -1,57 +1,32 @@
 const expect = require('chai').expect;
 
-const MajorColors = [
-    "white",
-    "red",
-    "black",
-    "yellow",
-    "violet"
-];
 
-const MinorColors = [
-    "blue",
-    "orange",
-    "green",
-    "brown",
-    "slate"
-];
+var get_Reference = require("./get_Color_Pair.js");
 
-function GetColorFromPairNumber(pairNumber) {
-    let colorPair = {};
-    const zeroBasedPairNumber = pairNumber - 1;
-    const majorColorIndex =
-        Math.floor(zeroBasedPairNumber / MinorColors.length);
-    colorPair.major = MajorColors[majorColorIndex];
-    colorPair.minor = MinorColors[zeroBasedPairNumber % MinorColors.length];
-    return colorPair
-}
-
-function GetPairNumberFromColors(majorColor, minorColor) {
-    let majorIndex = 0;
-    let minorIndex = 0;
-    for(majorIndex = 0; majorIndex < MajorColors.length; majorIndex++) {
-        if(MajorColors[majorIndex] == majorColor) {
-            break;
-        }
+function ToString()
```

Comment:
The task is to get the reference manual as a string, not to output to the console. Make the function return a string with the formatted output